app.js

```diff
@@ -60,23 +75,202 @@ nameForm.addEventListener('submit', (event) => {
-    switch(randomNumber) {
-        case 0:
-            appState.computerSelectedItem = paper;
-            computerElements.paperElement.classList.add('item-visible');
-            break;
-        case 1:
-            appState.computerSelectedItem = rock;
-            computerElements.rockElement.classList.add('item-visible')
-            break;
-        default:
-            appState.computerSelectedItem = scissors;
-            computerElements.scissorsElement.classList.add('item-visible');
-    }
+    disablingBtn()
+    loader.classList.add('item-visible');
+
+    setTimeout( () => {
+        loader.classList.remove('item-visible');
+
+        const randomNumber = getRandom();
+        switch(randomNumber) {
+            case 0:
+                appState.computerSelectedItem = paper;
+                computerElements.paperElement.classList.add('item-visible');
+
+                if(appState.userSelectedItem === paper) {
+                    deadHeat();
+                } else if(appState.userSelectedItem === rock) {
+                    appState.points.computer++;
+                    computerPoints.innerHTML = appState.points.computer;
+                    computerWonRound();
+                } else {
+                    appState.points.user++;
+                    userPoints.innerHTML = appState.points.user;
+                    userWonRound();
+                }
+                break;
+            case 1:
+                appState.computerSelectedItem = rock;
+                computerElements.rockElement.classList.add('item-visible')
+
+                if(appState.userSelectedItem === rock) {
+                    deadHeat();
+                } else if(appState.userSelectedItem === paper) {
+                    appState.points.user++;
+                    userPoints.innerHTML = appState.points.user;
+                    userWonRound();
+                } else {
+                    appState.points.computer++;
+                    computerPoints.innerHTML = appState.points.computer;
+                    computerWonRound();
+                }
+                break;
+            default:
+                appState.computerSelectedItem = scissors;
+                computerElements.scissorsElement.classList.add('item-visible');
+
+                if(appState.userSelectedItem === scissors) {
+                    deadHeat();
+                } else if(appState.userSelectedItem === rock) {
+                    appState.points.user++;
+                    userPoints.innerHTML = appState.points.user;
+                    userWonRound();
+                } else {
+                    appState.points.computer++;
+                    computerPoints.innerHTML = appState.points.computer;
+                    computerWonRound();
+                }
+        }
+        
+    }, 1500);
+}
+
+function showNextRoundStatement(msg) {
+    roundModal.modal.classList.remove('invisible');
+    roundModal.msg.innerHTML = msg;
+}
+
+function deadHeat() {
```

Comment:
you could do a one function, which get a message as an argument instead of 3 functions (`deadHeat`, `userWonRound`, computerWonRound`) - they are literally the same, only message makes difference